website/client-old/js/controllers/settingsCtrl.js

```diff
@@ -164,14 +164,17 @@ habitrpg.controller('SettingsCtrl',
     $scope.restore = function(){
       var stats = $scope.restoreValues.stats,
         achievements = $scope.restoreValues.achievements;
-      User.set({
-        "stats.hp": stats.hp,
-        "stats.exp": stats.exp,
-        "stats.gp": stats.gp,
-        "stats.lvl": stats.lvl,
-        "stats.mp": stats.mp,
-        "achievements.streak": achievements.streak
-      });
+      if (_validateValues(stats.lvl)) {
```

Comment:
1) this method is called `_validateValues` but only validates a single value, level.

2) Wouldn't this be cleaner:

```
if (stats.lvl < 1) {
  Notification.error(env.t('invalidLevel'), true);
  return;
}

User.set({ /* stuff */ });
```